### DISCUSSION TITLE
installation on tails

### DISCUSSION THREAD

**Discussion by bedr8m** [akiraux/Akira#717] [Q&A]
hi, im trying to install akira on tails but i can't install it from synaptic package manager, I can't install it from terminal following the installation instructions. if I download the deb file and try to install it from terminal, I don't succeed. is it possible to install akira on tails? I have been fiddling with it for hours, but to no avail.

**Comment by Alecaddd** [comment]
I never used Tails so I don't know its limitations.
If you can't compile Akira from source, the other option is installing the flatpak, but I don't know if that's supported.
Do you have any insights on what it's needed to make an application available for tails?
I don't currently release it via synaptic as I didn't create a PPA or launchpad for it, if that's even the way to go.
